src/Api/SecretsManager/Models/Response/ProjectServiceAccountsAccessPoliciesResponseModel.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Bit.Core.Models.Api;
+using Bit.Core.SecretsManager.Entities;
+
+namespace Bit.Api.SecretsManager.Models.Response;
+
+public class ProjectServiceAccountsAccessPoliciesResponseModel : ResponseModel
+{
+    private const string _objectName = "serviceAccountAccessPolicies";
```

Comment:
```suggestion
    private const string _objectName = "projectServiceAccountsAccessPolicies";
```

I would keep the object names as close to the model as possible.